pagespeed/apache/mod_instaweb.cc

```diff
@@ -1789,6 +1790,8 @@ static const command_rec mod_pagespeed_filter_cmds[] = {
   APACHE_CONFIG_OPTION(kModPagespeedBlockingRewriteRefererUrls,
                        "wildcard_spec for referer urls which trigger blocking "
                        "rewrites"),
+  APACHE_CONFIG_OPTION(kModPagespeedFollowFlushes,
+      "Attempt to mirror incoming flushes for html streams in the output"),
```

Comment:
Better to just give the option a name when registering this, then no code in the file is needed for mod_pagespeed to pick it up.
(Also it won't be inconsistent on whether it's directory vs. global or whatever.)